sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -666,7 +667,7 @@ public String getId() {
     }
 
     private <T> FeedResponse<T> prepareFeedResponse(
-        FeedResponse<Document> response,
+        FeedResponse<T> response,
         boolean isChangeFeed,
         Class<T> classType) {
```

Comment:
the classType does not seem to be used any more

Reply:
Fixed in next iteration